react/features/status-bar/components/BaseStatusIcon.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+
+import UIUtil from '../../../../modules/UI/util/UIUtil';
```

Comment:
I could focus on the z-indexing but there would still be cutoff in vertical filmstrip due to overflow hidden with longer tooltips.